spec/release_spec.rb

```diff
@@ -379,4 +379,23 @@ def load_release(release_data)
       end
     end
   end
+
+  context "correct checkout behavior:" do
+    let(:repo) { extracted_asset_dir("foo", "foo-boshrelease-repo.zip") }
+
+    describe "#update_repo_with_ref" do
+      subject { Rugged::Repository.new(File.join(releases_dir, name)) }
+      context "calls checkout_tree and checkout" do
+        before do
+          system("rm -rf #{releases_dir}")
```

Comment:
Could you use [FileUtils.rmdir](http://ruby-doc.org/stdlib-2.2.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-rmdir) instead?